example/example.cpp

```diff
@@ -24,507 +24,627 @@ using namespace isaac;
 #define VOLUME_Y 64
 #define VOLUME_Z 64
 
+#define PARTICLE_VOLUME_X 64
+#define PARTICLE_VOLUME_Y 64
+#define PARTICLE_VOLUME_Z 64
+
+#define PARTICLE_COUNT 64
+
+
+
 //////////////////////
 // Example Source 1 //
 //////////////////////
+/*
+ * First volume source
```

Comment:
please remove this comment and rename the comment above `Volume Source 1` else both comments will somehow say the same.